### DISCUSSION TITLE
How do you mark a checkbox as checked when using wireui

### DISCUSSION THREAD

**Discussion by happymacarts** [wireui/wireui#659] [Q&A]
I am trying to figure this out and am stumped
will make the checkbox disappear
```<x-ct-checkbox id="right-label" label="Allow SMS*" @if($mobile_sms) checked @endif />```
renders as ```<x-ct-checkbox id="right-label" label="Allow SMS*" checked=""></x-ct-checkbox>```

``` 
<x-ct-checkbox id="right-label" label="Allow SMS*" :checked="$work_sms"/>
``` 

will always render checked even if value is false
```
<input type="checkbox" class="form-checkbox rounded transition ease-in-out duration-100 
  border-secondary-300 text-primary-600 focus:ring-primary-600 focus:border-primary-400
  dark:border-secondary-500 dark:checked:border-secondary-600 dark:focus:ring-secondary-600
  dark:focus:border-secondary-500 dark:bg-secondary-600 dark:text-secondary-600
  dark:focus:ring-offset-secondary-800" id="right-label" checked="False">
```

What is the correct way to render as checked?

**Comment by Fludem** [comment]
<x-ct-checkbox id="right-label" label="Allow SMS*" wire:model="work_sms"/>

The better way to make it checked is also adding 'checked' at the end, you are rendering 

What you are doing is binding checkeds value to work_sms so like if work_sms is false, it'll put false, if its true it'll put true

the better way to make a box checked in html is to either have checked at the end or not

Like this

<input type="checkbox" name"work_sms" checked> 
That is a checked checkbox
<input type="checkbox" name"work_sms"> is not checked

**Comment by Fludem** [comment]
If possible try binding it to your model using wire:model="work_sms"
